src/client_side_reply.cc

```diff
@@ -416,16 +400,16 @@ clientReplyContext::sendClientOldEntry()
  * IMS request to revalidate a stale entry.
  */
 void
-clientReplyContext::handleIMSReply(StoreIOBuffer result)
+clientReplyContext::handleIMSReply(const StoreIOBuffer result)
```

Comment:
Done in commit fe6e0ed. Normally, I resist fixing constness when the declaration does not need to be changed for other reasons, but I think these two cases warrant an exception because a StoreIOBuffer result change is a red flag that requires additional scrutiny (and some were buggy during this branch development); it is trivial to miss such changes in these long/complex methods.